### DISCUSSION TITLE
Is there a way to access Vertices and normal information

### DISCUSSION THREAD

**Discussion by StefanThoene** [ansys/pyprimemesh#777] [Q&A]
Dear PyAnsys team i am trying to use pyprimemesh to create a mesh and i need to access the vertices and normals of my created surface mesh is there a way to directly access them?
i couldn't find that information in the documentation or in an example.

**Comment by RobPasMue** [comment]
Hi @StefanThoene - thanks for opening this discussion! Redirecting it to @ansys/pyprimemesh team and repository

**Reply by StefanThoene** [reply]
Thank you @RobPasMue

**Reply by hlee0122** [reply]
Hi @StefanThoene, am I correct that you're trying to get the orientation info of the surfaces at given vertices?

**Reply by StefanThoene** [reply]
@hlee0122 yes that is correct, and i would like to acces the vertex coordinate and corresponding surface normal
